pom.xml

```diff
@@ -77,6 +68,10 @@
       <!-- Nexus Staging -->
       <build>
         <plugins>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.11.0</version>
+          </plugin>
```

Comment:
the default version was 3.1.  `<maven.compiler.release>` requires 3.6 or later..